### PR TITLE
Adding new graph worflows for Hypervisor deployment via out of band c…

### PR DESCRIPTION
**Pull request by rahmanmuhamad**
…ontroller , this is the initial pull request and will be followed by few more on other modules

### CODE REVIEW

File: lib/graphs/dell-wsman-os-deployment-create-graph.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// Copyright 2016, EMC, Inc.
```
Comment (lanchongyizu) [inline lib/graphs/dell-wsman-os-deployment-create-graph.js]: //Copyright 2017, Dell EMC, Inc.  The same problem exists in dell-wsman-os-deployment-deploy-graph.js.

File: lib/graphs/dell-wsman-os-deployment-create-graph.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+         {
+            label: 'dell-wsman-os-create',
+            taskName: 'Task.Dell.Wsman.Os.Create'
+        }
+    ]
+};
+
+
+
+
```
Comment (lanchongyizu) [inline lib/graphs/dell-wsman-os-deployment-create-graph.js]: Please remove the empty lines. The same problem exists in dell-wsman-os-deployment-deploy-graph.js.

**Comment by lanchongyizu** [comment]
@rahmanmuhamad Could you please help to add the unit test for both graphs?

**Comment by nortonluo** [comment]
This graph depends on https://github.com/RackHD/on-tasks/pull/570

**Pull request merged**